docs/getting-started/quickstart.md

```diff
@@ -1,34 +1,38 @@
 # Quickstart
 
-There are a couple of ways to get started with n8n.
+You can run n8n both locally and remotely. There are three different ways in which you can get started with n8n.
 
 [[toc]]
 
-## Sign-up on n8n.cloud
+# Run n8n locally
```

Comment:
I think we should list n8n cloud first because it can be run with least friction. Also, we should avoid using h1 within the content as it messes up the toc.